code/game/machinery/bots/farmbot.dm

```diff
@@ -55,6 +55,11 @@
 	var/path[] = new() // used for pathing
 	var/frustration
 
+/obj/machinery/bot/farmbot/vox_garden_farmbot
+	name = "Special vox trader farmbot"
```

Comment:
Vox is a proper noun

Reply:
Alright just tell me how you want it and i'll change it.

Reply:
Either way is fine, either special Vox trader farmbot or Special Vox Trader Farmbot

Reply:
Why would vox be a proper noun? Human isn't

Reply:
maybe I'm crazy, I thought we treated races as proper nouns, because they are like a demonym. For example, "Pinoy" is a proper noun.

Human isn't a proper noun (right now) because it would describe everyone, but in a future where "human" refers to a specific demographic group, it would be a proper noun, right?

Reply:
Human and vox are both species, like dog and cat